subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorParallelIntegrationTest.groovy

```diff
@@ -698,6 +700,10 @@ class WorkerExecutorParallelIntegrationTest extends AbstractWorkerExecutorIntegr
         succeeds("allTasks")
     }
 
+    @Requires(
+        value = IntegTestPreconditions.NotConfigCached,
+        reason = "Assumptions about project locking introduced in https://github.com/gradle/gradle/pull/2171 do not hold"
```

Comment:
⛏️ Perhaps something about the incompatible behavior here could help future readers: "Tasks run in parallel when cc is enabled"